src/content/en/2021/capabilities.md

```diff
@@ -1,20 +1,234 @@
 ---
 #See https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide#metadata-to-add-at-the-top-of-your-chapters
 title: Capabilities
-description: TODO
-authors: []
-reviewers: []
-analysts: []
+description: Capabilities give web apps access to hardware interfaces, enhance web-based productivity apps, and more.
+authors: [christianliebel]
+reviewers: [tomayac, hemanth]
+analysts: [tomayac]
```

Comment:
Reminder to copy these people's details to `src/config/2021.json` as part of this PR. All were involved last year so should just be a copy and paste from that and an update to the roles.